torch_brain/models/poyo.py

```diff
@@ -346,9 +329,9 @@ def __call__(self, data: Data) -> Dict:
             "latent_index": latent_index,
             "latent_timestamps": latent_timestamps,
             # output sequence
-            "output_session_index": pad8(session_index),
+            "output_session_index": pad8(output_session_index),
             "output_timestamps": pad8(output_timestamps),
-            "output_mask": pad8(output_mask),
+            "output_mask": track_mask8(output_session_index),
```

Comment:
I went with the `pad8` route because that way we can `&` the evaluation mask with it (if doing eval) and don't have to pass a separate eval mask to the validator.